index.css

```diff
@@ -212,4 +214,54 @@ footer {
 
 footer .license {
 	float: right;
-}
\ No newline at end of file
+}
+
+footer br{
+	display: none;
+}
+
+h2{
+	text-align: center;
+}
+
+@media only screen and (max-width: 77em){
```

Comment:
Where did 77 em come from?

Reply:
78em is the width till which 3 sections can be adjusted , because I have styled the width of each section as 26em each and the main as 78em for larger device width . All media queries are considered by the max sections the particular width can hold. They are not just random numbers.

Reply:
I would understand 78 em, that would make sense. I don't understand 77 em.

Reply:
Changed it 78em in the next commit.